src/pages/guides/getting-started/editing-the-document/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Editing the document
+description: Make edits to your Photoshop documents with your UXP-based plugin
+contributors:
+  - https://github.com/pklaschka
+---
+
+# Editing the document
```

Comment:
Can we replace 'document' with 'Ps document', so that it becomes absolutely clear to the developer what are they editing?

Reply:
Agreed, I'll change it (the new title probably won't fit nicely into the sidebar -> it'll have to stay as it is over there, but we can at least adjust it on the page itself)